ClavaLaraApi/src-lara-clava/clava/clava/opt/NormalizeToSubset.js

```diff
@@ -0,0 +1,27 @@
+laraImport("weaver.Query");
+laraImport("clava.code.StatementDecomposer");
+laraImport("clava.pass.SimplifyLoops");
+laraImport("clava.pass.DecomposeVarDeclarations");
+laraImport("clava.pass.DecomposeDeclStmt");
+laraImport("clava.pass.SimplifySelectionStmts");
+laraImport("clava.code.SimplifyAssignment");
+
+function NormalizeToSubset($startJp) {
+  const declStmt = new DecomposeDeclStmt();
+  const varDecls = new DecomposeVarDeclarations();
+  const statementDecomposer = new StatementDecomposer();
+  const simplifyLoops = new SimplifyLoops(statementDecomposer);
+  const simplifyIfs = new SimplifySelectionStmts(statementDecomposer);
+
+  simplifyLoops.apply($startJp);
+  simplifyIfs.apply($startJp);
+
+  declStmt.apply($startJp);
+  varDecls.apply($startJp);
```

Comment:
I see here a possible advantage of having functions over classes, it would simplify this kind of code when written like this. But I was wondering if we could have helper methods where we could specify something like:

```
const statementDecomposer = new StatementDecomposer();
const passes = [new SimplifyLoops(statementDecomposer), new SimplifySelectionStmts(statementDecomposer), DecomposeDeclStmt, DecomposeVarDeclarations];
Passes.apply(passes, $startJp);
```
Would this be an improvement? How would it look with functions instead of classes?